packages/coverage-contract/src/coverage-contract-types/factories/CoverageContractAbi__factory.ts

```diff
@@ -0,0 +1,451 @@
+        type: "bool",
+        components: null,
+      },
+    ],
+    name: "echo_bool",
+    outputs: [
+      {
+        name: "",
+        type: "bool",
+        components: null,
+      },
+    ],
+  },
+  {
+    type: "function",
+    inputs: [
+      {
+        name: "input",
+        type: "b256",
+        components: null,
+      },
+    ],
+    name: "echo_b256",
+    outputs: [
+      {
+        name: "",
+        type: "b256",
+        components: null,
+      },
+    ],
+  },
+  {
+    type: "function",
+    inputs: [
+      {
+        name: "input",
+        type: "str[1]",
+        components: null,
+      },
+    ],
+    name: "echo_str_1",
+    outputs: [
+      {
+        name: "",
+        type: "str[1]",
+        components: null,
+      },
+    ],
+  },
+  {
+    type: "function",
+    inputs: [
+      {
+        name: "input",
+        type: "str[2]",
+        components: null,
+      },
+    ],
+    name: "echo_str_2",
+    outputs: [
+      {
+        name: "",
+        type: "str[2]",
+        components: null,
+      },
+    ],
+  },
+  {
+    type: "function",
+    inputs: [
+      {
+        name: "input",
+        type: "str[3]",
+        components: null,
+      },
+    ],
+    name: "echo_str_3",
+    outputs: [
+      {
+        name: "",
+        type: "str[3]",
+        components: null,
+      },
+    ],
+  },
+  {
+    type: "function",
+    inputs: [
+      {
+        name: "input",
+        type: "str[8]",
+        components: null,
+      },
+    ],
+    name: "echo_str_8",
+    outputs: [
+      {
+        name: "",
+        type: "str[8]",
+        components: null,
+      },
+    ],
+  },
+  {
+    type: "function",
+    inputs: [
+      {
+        name: "input",
+        type: "str[9]",
+        components: null,
+      },
+    ],
+    name: "echo_str_9",
+    outputs: [
+      {
+        name: "",
+        type: "str[9]",
+        components: null,
+      },
+    ],
+  },
+  {
+    type: "function",
+    inputs: [
+      {
+        name: "input",
+        type: "(u8, u8)",
+        components: null,
+      },
+    ],
+    name: "echo_tuple_u8",
+    outputs: [
+      {
+        name: "",
+        type: "(u8, u8)",
+        components: null,
+      },
+    ],
+  },
+  {
+    type: "function",
+    inputs: [
+      {
+        name: "input",
+        type: "(u64, u64)",
+        components: null,
+      },
+    ],
+    name: "echo_tuple_u64",
+    outputs: [
+      {
+        name: "",
+        type: "(u64, u64)",
+        components: null,
+      },
+    ],
+  },
+  {
+    type: "function",
+    inputs: [
+      {
+        name: "input",
+        type: "(bool, u64)",
+        components: null,
+      },
+    ],
+    name: "echo_tuple_mixed",
+    outputs: [
+      {
+        name: "",
+        type: "(bool, u64)",
+        components: null,
+      },
+    ],
+  },
+  {
+    type: "function",
+    inputs: [
+      {
+        name: "input",
+        type: "[u8; 2]",
+        components: null,
```

Comment:
same comment about tuples - I think arrays have components now and this should be regenerated.